docs/content/users/quickstart.md

```diff
@@ -70,6 +70,41 @@ Please note that you will need to change the PHP version to 7.4 to be able to wo
     ddev launch
     ```
 
+## Contao
+
+Further information can also be found in the [Contao documentation](https://docs.contao.org/manual/en/guides/local-installation/ddev/).
+
+=== "Composer"
+
+    ```bash
+    mkdir -p ~/projects/my-contao && cd ~/projects/my-contao
+    ddev config --project-type=php --docroot=public --webserver-type=apache-fpm --php-version=8.2 --create-docroot --timezone=Europe/Berlin
+    ddev composer create contao/managed-edition:5.3
+    
+    # After installation, the database access data must be entered in the .env.local file. At the same time, we also set up Mailpit here too:
+    # DATABASE_URL=mysql://db:db@db:3306/db
+    # MAILER_DSN=smtp://localhost:1025
+    
+    # Create the database
+    ddev exec "bin/console contao:migrate"
+    
+    # Create backend user
+    ddev exec "bin/console contao:user:create"
+    ddev launch
```

Comment:
```suggestion
    ddev launch /contao
```
would it make sense to point the ddev launch command to the login page for the backend. with just `ddev launch` i am getting to the front end and that leaves me with a 404 following these instructions
![Screenshot 2024-11-03 at 19 45 54](https://github.com/user-attachments/assets/b657218c-8954-4a08-8e48-3efa91a7b0a7)

Reply:
Thanks. Added this as a comment.

Reply:
I suggest to leave only `ddev launch /contao` because it's the only page that works from the start. (And Contao Manager also redirects to the admin area first.)